core/src/test/scala/org/apache/spark/sql/delta/EvolvabilitySuite.scala

```diff
@@ -70,7 +70,13 @@ class EvolvabilitySuite extends EvolvabilitySuiteBase with DeltaSQLCommandTest {
 
   testQuietly("parse old version CheckpointMetaData") {
     assert(JsonUtils.mapper.readValue[CheckpointMetaData]("""{"version":1,"size":1}""")
-      == CheckpointMetaData(1, 1, None))
+      === CheckpointMetaData(1, 1, None, None, None))
+  }
+
+  test("parse partial version CheckpointMetaData") {
+    assert(JsonUtils.mapper.readValue[CheckpointMetaData](
```

Comment:
nit: could you use `parts` rather than `sizeInBytes` in this test so that we are using a real test case?